openslides_backend/action/util/crypto.py

```diff
@@ -0,0 +1,15 @@
+import secrets
+
+RANDOM_STRING_CHARS = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+
+def get_random_string(length: int, allowed_chars: str = RANDOM_STRING_CHARS) -> str:
+    """
+    Return a securely generated random string.
+    The bit length of the returned value can be calculated with the formula:
+        log_2(len(allowed_chars)^length)
+    For example, with default `allowed_chars` (26+26+10), this gives:
+      * length: 12, bit length =~ 71 bits
+      * length: 22, bit length =~ 131 bits
```

Comment:
You can remove this part of the docstring, we don't care about the bit length

Reply:
I deleted this comment lines.